app/submitter/submitter.py

```diff
@@ -141,3 +143,46 @@ def send_message(self, message, tx_id, questionnaire_id, case_id=None):
         finally:
             if connection:
                 self._disconnect(connection)
+
+
+class GCSFeedback:
+    def __init__(self, bucket_name):
+        client = storage.Client()
+        self.bucket = client.get_bucket(bucket_name)
+        self.bucket.retention_period = 86400
+
+    def upload(self, schema, feedback_data, session_data):
+        object_key = str(uuid4())
+        blob = self.bucket.blob(object_key)
+        blob.metadata = {
+            "feedback_count": session_data.feedback_count + 1,
+            "form_type": schema.form_type,
+            "language_code": session_data.language_code,
+            "object_key": object_key,
+            "region_code": schema.region_code,
+            "submitted_at": session_data.submitted_time,
```

Comment:
The card specified the feedback submitted time not the questionnaire submitted time. However, I am not sure we even want/need submitted time as all blobs have a built-in [timeCreated](https://cloud.google.com/storage/docs/json_api/v1/objects) value. I am not sure if big query can use that or whether it needs explicit timestamp data.